gsoc/project_NOvA.md

```diff
@@ -0,0 +1,8 @@
+---
+project: NOvA
+layout: default
+description: |
+[NOvA](https://www-nova.fnal.gov) is a neutrino oscillation experiment whose measurement of this particle's properties relies on efficient identification and robust reconstruction of events. NOvA has already shown the first use of CNNs (convolutional neural networks) for [event classification] (http://inspirehep.net/record/1444342) in a HEP result. Now we are expanding the program of machine learning applications on NOvA events for their use in full event reconstruction and eventually event simulation applications.
```

Comment:
This line must be indented (2 spaces) else it is not recognized by Jekyll... Ideally you would like to split this very long line into multiple lines all indented the same way...